XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Utilities/WindowsMixedRealityUtilities.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if WINDOWS_UWP
+#if ENABLE_DOTNET
+using System;
+#endif // ENABLE_DOTNET
+using System.Runtime.InteropServices;
+using UnityEngine.XR.WSA;
+using Windows.Perception.Spatial;
+#endif // WINDOWS_UWP
+
+namespace XRTK.WindowsMixedReality.Utilities
+{
+    public static class WindowsMixedRealityUtilities
+    {
+#if WINDOWS_UWP
+#if ENABLE_DOTNET
```

Comment:
I've never seen/used `ENABLE_DOTNET` before

Reply:
It's the counterpart of `ENABLE_IL2CPP` and a relict from a past time when there was a .NET and Il2CPP scripting backend. This can go. Thanks.